### PR TITLE
Add blog post for 4.4 dev1

### PR DESCRIPTION
**Pull request by clayjohn**
Socials stuff:
- URL: https://godotengine.org/article/dev-snapshot-godot-4-4-dev-1/
- TL;DR: The merge window for 4.4 reopened with the 4.3 release, and a lot of goodies had been queued. More than 200 changes were already merged and ready to test, including a Metal renderer for Apple devices, 3D physics interpolation, bicubic sampling for lightmaps, and many performance optimizations.
- Featured game:
  * Gourdlets, by AuntyGames
  * https://store.steampowered.com/app/2013730/Gourdlets/
  * https://www.auntygames.com/
  * https://x.com/auntygames
  * Screenshot: https://shared.akamai.steamstatic.com/store_item_assets/steam/apps/2013730/ss_1de975a26b2f43778aff647ca1d5772c5c63d64e.jpg?t=1723741519

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-4-dev-1.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.4.dev1/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+-----
+
+*The cover illustration is from* [**TODO**](TODO), * by [TODO](https://TODO).*
+
+## Highlights
+
+After only a week of development we are bringing you a bunch of fixes and improvements
```
Comment (Calinou) [inline collections/_article/dev-snapshot-godot-4-4-dev-1.md]:
```suggestion
After only a week of development, we are bringing you a bunch of fixes and improvements:
```

File: collections/_article/dev-snapshot-godot-4-4-dev-1.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+### Metal Rendering Backend
+
+We hinted that a Metal backend was in the works and it is finally here ([GH-88199](https://github.com/godotengine/godot/pull/88199))! 
+
+Metal is a low-level graphics API similar to Vulkan or D3D12. Currently Godot supports using Vulkan or D3D12, but
+neither of those are available on MacOS and iOS, so we use a library called [MoltenVK](https://github.com/KhronosGroup/MoltenVK)
+to run Vulkan over Metal. MoltenVK is great and has worked well for us. But having our own Metal implementation is more efficient
+and allows us to have a greater control over what features we support and what performance tradeoffs we make.
```
Comment (Calinou) [inline collections/_article/dev-snapshot-godot-4-4-dev-1.md]:
```suggestion
Metal is a low-level graphics API similar to Vulkan or D3D12. Godot currently supports using Vulkan or D3D12, but
neither of those are available on MacOS and iOS, so we use a library called [MoltenVK](https://github.com/KhronosGroup/MoltenVK)
to run Vulkan over Metal. MoltenVK is great and has worked well for us. However, having our own Metal implementation is more efficient
and allows us to have a greater control over what features we support and what performance tradeoffs we make. 
```

File: collections/_article/dev-snapshot-godot-4-4-dev-1.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+Metal is a low-level graphics API similar to Vulkan or D3D12. Currently Godot supports using Vulkan or D3D12, but
+neither of those are available on MacOS and iOS, so we use a library called [MoltenVK](https://github.com/KhronosGroup/MoltenVK)
+to run Vulkan over Metal. MoltenVK is great and has worked well for us. But having our own Metal implementation is more efficient
+and allows us to have a greater control over what features we support and what performance tradeoffs we make. 
+
+Earlier results show that the Metal backend on MacOS is at least as fast as the Vulkan backend and in many cases, much faster.
+
+Right now we only support Metal on Apple silicon devices. That includes all iOS devices and the M1/M2/M3 Macs. Other
+Apple devices will continue to work with the MoltenVK backend. Currently this limitation is simply due to the fact that
+few contributors working in this area have access to non-Apple Silicon devices.
```
Comment (Calinou) [inline collections/_article/dev-snapshot-godot-4-4-dev-1.md]:
```suggestion
Right now, we only support Metal on Apple Silicon (ARM) devices. That includes all iOS devices and the M1/M2/M3 Macs. Intel-based
Apple devices will continue to work with the MoltenVK backend. Currently, this limitation is due to the fact that
few contributors working in this area have access to non-Apple Silicon devices. 
```

File: collections/_article/dev-snapshot-godot-4-4-dev-1.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+</video>
+
+### Lightmap Bicubic sampling
+Bicubic sampling is a method for reading from a lightmap that smooths our sharp edges. It is especially useful when
+you bake lightmaps with shadows at a low texture resolution. We supported this in Godot 3, and have now brought it back in
+[GH-89919](https://github.com/godotengine/godot/pull/89919).
+
+![Comparison between shadows with bicubic on and off](/storage/releases/4.4-dev1/bicubic-compare.webp)
+
+Bicubic sampling does come with a small run-time performance cost. So it can be disabled in a project setting if needed.
```
Comment (Calinou) [inline collections/_article/dev-snapshot-godot-4-4-dev-1.md]:
```suggestion
Bicubic sampling comes with a small run-time performance cost on the GPU, so it can be disabled in a project setting if needed.
```

File: collections/_article/dev-snapshot-godot-4-4-dev-1.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+- [Core] Optimize `String.join` ([GH-92550](https://github.com/godotengine/godot/pull/92550)).
+- [Core] Optimize `String.replace` ([GH-92546](https://github.com/godotengine/godot/pull/92546)).
+- [Import] Disable normal raycaster by default ([GH-93727](https://github.com/godotengine/godot/pull/93727)). This makes
+    importing meshes with LODs approximately 2X faster. We expect that the quality of imported LODs will be the same.
+    If you notice any issues, please let us know. You can re-enable the raycaster in the object's import settings under
+    `lods/raycast_normals`.
+- [Editor] Optimize editor grid recalculation ([GH-92734](https://github.com/godotengine/godot/pull/92734)).
+- [Editor] Optimize closing the editor settings ([GH-95704](https://github.com/godotengine/godot/pull/95704)).
+- [Rendering] Optimize meshes for vertex cache ([GH-94241](https://github.com/godotengine/godot/pull/94241)). In scenes
+    with a lot of geometry this can result in a 20% improvement in frame time.
```
Comment (Calinou) [inline collections/_article/dev-snapshot-godot-4-4-dev-1.md]:
```suggestion
    with a lot of geometry, this can result in a 20% improvement in frame time. 
```

File: collections/_article/dev-snapshot-godot-4-4-dev-1.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+### And more!
+
+There are too many exciting changes to list them all here, but here are a few more. 
+
+- [Editor] Fix `uid://` paths fail to load at editor startup ([GH-95689](https://github.com/godotengine/godot/pull/95689)).
+- [Editor] Correct rotation gizmo plane math for off-center perspective view ([GH-95402](https://github.com/godotengine/godot/pull/95402)).
+- [Editor] Make viewport grid visible on all three planes in ortho camera view ([GH-93869](https://github.com/godotengine/godot/pull/93869)).
+- [Editor] Expose GDScript syntax highlighter to editor plugins ([GH-95764](https://github.com/godotengine/godot/pull/95764)).
+- [Rendering] Fix generated light probes placing too close to manual light probes ([GH-83497](https://github.com/godotengine/godot/pull/83497)).
+- [Rendering] Antialias direct light samples in LightmapperRD ([GH-95828](https://github.com/godotengine/godot/pull/95828)).
```
Comment (Calinou) [inline collections/_article/dev-snapshot-godot-4-4-dev-1.md]: I feel this one is worth mentioning in the bicubic sampling paragraph as well, given how they tend to work together to provide the best results.

Comment (akien-mga) [reply collections/_article/dev-snapshot-godot-4-4-dev-1.md]: Added.

**Comment by akien-mga** [comment]
Thanks!
